OpenSim/Simulation/Model/Muscle.cpp

```diff
@@ -96,6 +93,33 @@ void Muscle::updateFromXMLNode(SimTK::Xml::Element& aNode, int versionNumber)
             }
             XMLDocument::renameChildNode(aNode, "pennation_angle", "pennation_angle_at_optimal");
         }
+        if (versionNumber <= 30513) {
+            SimTK::Xml::element_iterator minControlElt =
+                aNode.element_begin("min_control");
+            double minControl = 0;
+            if (minControlElt != aNode.element_end()) {
+                minControlElt->getValueAs<double>(minControl);
+
+                // remove the min_control property in XML since it is a result
+                // of a mistake with the previous versions not updating
+                // the min_control to reflect the min_activation. Removing it
+                // allows the Muscle to use the default that is specified by the
+                // derived concrete Muscle
```

Comment:
Could you clarify the comment to say that the property is only removed if it has the value 0?